src/variables/table.tsx

```diff
@@ -0,0 +1,166 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import { ArrayExt } from '@lumino/algorithm';
+
+import { ReactWidget } from '@jupyterlab/apputils';
+
+import React, { useEffect, useState } from 'react';
+
+import { VariablesModel } from './model';
+
+import { convertType } from '.';
+
+import { IDebugger } from '../tokens';
+
+import { CommandIDs } from '..';
+
+import { CommandRegistry } from '@lumino/commands';
+
+/**
+ * The body for Table Variables .
+ */
+export class VariablesBodyTable extends ReactWidget {
+  /**
+   * Instantiate a new Body for the Table Variables .
+   * @param model The model for the variables.
+   */
+  constructor(options: VariablesBodyTable.IOptions) {
+    super();
+    this._model = options.model;
+    this._commands = options.commands;
+    this.addClass('jp-DebuggerVariables-body');
```

Comment:
Should this call to `addClass` be placed at the end of the constructor, to follow the same pattern as for the other components?